tests/gwtests.csv

```diff
@@ -0,0 +1,15 @@
+action,ms,bytes,lang,code,repeat,minver,comment
+before,0,0,q,".proc.loadf getenv[`KDBCODE],"/gateway/gatewaylib.q"",1,,
```

Comment:
I think as long as you use `-proctype gateway` when running the tests this should be automatically loaded, no need to load explicitly IMO